charts/datadog/values.yaml

```diff
@@ -605,19 +605,16 @@ datadog:
   ## Enable process agent and provide custom configs
   processAgent:
     # datadog.processAgent.enabled -- Set this to true to enable live process monitoring agent
-
-    ## Note: /etc/passwd is automatically mounted when `processCollection` or `processDiscovery` is enabled.
+    # DEPRECATED. Set `processCollection` or `containerCollection` instead.
+    ## Note: /etc/passwd is automatically mounted when `processCollection`, `processDiscovery`, or `containerCollection` is enabled.
     ## ref: https://docs.datadoghq.com/graphing/infrastructure/process/#kubernetes-daemonset
     enabled: true
 
     # datadog.processAgent.processCollection -- Set this to true to enable process collection in process monitoring agent
```

Comment:
Should we remove `in process monitoring agent` from the comment here?

Reply:
Done